src/task-definitions.js

```diff
@@ -266,6 +248,14 @@ tasks.exportNotebook = new UserTask({
   callback() { dispatcher.exportNotebook() },
 })
 
+tasks.exportNotebookAsReport = new UserTask({
+  title: 'Export Notebook as report',
+  keybindings: ['shift+e'],
```

Comment:
Could we make this keybinding `meta+shift+e`?

Reply:
Oh duh yes of course